sms-notifications/assets/index.html

```diff
@@ -63,6 +63,10 @@ <h3>Add Recipients</h3>
           <fieldset>
             <label for="newRecipientInput">New Recipient Phone Number:</label>
             <input type="tel" id="newRecipientInput" required />
+            <input id="addRecipientButton"
+                   class="button-primary"
+                   type="button"
```

Comment:
You could turn this into:
```suggestion
                   type="submit"
```
And then you don't have to set up a separate event handler in the JavaScript.

Reply:
Good catch! I force-pushed an amended commit with this change, and some of the associated complexity cleaned up.